### PR TITLE
Feature/link cubes without groups

### LINKED ISSUE #581: Links between data cubes
Data cubes are often driven by technical needs. We split data into different data cubes to get better performance, more granularity or we want to divide data set into logical parts. 

Right now when user want to switch data source, we present blank definition to them (with defaults defined in config). 

We want to give tools for cross data source data exploration. First idea is to preserve parts of view definition between data cubes.

Proposal: Left menu switches datasource but keeps view definition
- what parts of view definition should we keep? filters? splits? measures?
- we must ensure that incompatible definitions are omitted.
- we must check how resolvers will behave in this scenario - for example if we switch from double splitter table to datasource without split dimensions, we should get total viz.

Comments? @cedrics @alexbusu @michalmisiewicz @mkuthan @KarolakJakub

### PR DESCRIPTION
**Pull request by adrianmroz**
Closes #581

### CODE REVIEW

File: src/client/components/header-bar/header-bar.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export interface HeaderBarProps {
+  customization?: Customization;
+  title?: string;
+}
+
+export const HeaderBar: React.SFC<HeaderBarProps> = props => {
+  const { customization, title } = props;
+
+  let headerStyle: any = null;
+  if (customization && customization.headerBackground) {
```
Comment (erykpiast) [inline src/client/components/header-bar/header-bar.tsx]: I suppose with a ternary expression you could avoid `any`.

File: src/client/components/side-drawer/side-drawer.tsx [hunk 2]
```diff
@@ -99,8 +97,21 @@ export class SideDrawer extends React.Component<SideDrawerProps, SideDrawerState
     </div>;
   }
 
+  private cubeHref(dataCube: DataCube): string {
+    const { getCubeViewHash, essence } = this.props;
+    const { dataCube: currentCube } = essence;
+    if (!DataCube.isDataCube(dataCube)) return "";
+    const cleanCubeHref = `#${name}`;
+
+    if (dataCube === currentCube) return cleanCubeHref;
```
Comment (erykpiast) [inline src/client/components/side-drawer/side-drawer.tsx]: Wouldn't single `if` with `&&` be less surprising?

File: src/common/utils/rules/predicates.ts [hunk 3]
```diff
@@ -79,7 +79,7 @@ export class Predicates {
 
   public static noSelectedMeasures(): VisualizationIndependentPredicate {
     return ({ series }) => {
-      return series.count() === 0;
+      return series.isEmpty();
```
Comment (erykpiast) [inline src/common/utils/rules/predicates.ts]:
```
({ series }) => series.isEmpty();
```
would be enough :)

**Comment by adrianmroz** [comment]
> I can't wait for this feature on prod!

Don't nitpick then! :D

**Pull request merged**